newsroom/auth_server/oauth2.py

```diff
@@ -32,6 +33,14 @@
 
 @blueprint.route(TOKEN_ENDPOINT, methods=["POST"])
 def issue_token():
+    current_time = utcnow()
+
+    client_id = request.authorization.get("username")
+    if client_id:
+        client = get_cached_resource_by_id("oauth_clients", client_id)
+        superdesk.get_resource_service("oauth_clients").system_update(
+            ObjectId(client_id), {"last_active": current_time}, client
+        )
     return authorization.create_token_response()
```

Comment:
not sure if this can fail in general, but imo would be better to do it like:
```
try:
    return authorization.create_token_response()
else:
    log the time
```